velox/docs/functions/spark/array.rst

```diff
@@ -2,6 +2,31 @@
 Array Functions
 =============================
 
+.. spark:function:: aggregate(array(E), start, merge, finish) -> array(E)
```

Comment:
Do we have an implementation for this function? Is it even scalar function? Would you please double check?

Reply:
https://github.com/facebookincubator/velox/blob/main/velox/functions/sparksql/Register.cpp#L44
Yes, function aggregate was registered here.